conf_test.go

```diff
@@ -403,6 +403,28 @@ func TestHappyCasesFromConf(t *testing.T) {
 				qdb.WithAutoFlushInterval(1000),
 			},
 		},
+		{
```

Comment:
Please add some tests around sender behavior with disabled row-/time-based flush. You can use `TestRowBasedAutoFlush` and `TestTimeBasedAutoFlush` tests as a starting point.

Reply:
Just to be clear, in the file `http_sender_test.go` two more tests needs to be added:
- `TestNoFlushWhenAutoFlushRowsDisabled`
- `TestNoFlushWhenAutoFlushIntervalDisabled`

That will be similar to `TestNoFlushWhenAutoFlushDisabled`

Reply:
Yes, new tests should be added. They should verify that e.g. row-based flush still takes place when time-based flush is disabled.